synctl

```diff
@@ -292,11 +318,12 @@ def main():
             write("All processes exited; now restarting...")
 
     if action == "start" or action == "restart":
+        errors = 0
         if start_stop_synapse:
             # Check if synapse is already running
             if os.path.exists(pidfile) and pid_running(int(open(pidfile).read())):
                 abort("synapse.app.homeserver already running")
-            start(configfile, bool(options.daemonize))
+                errors += int(start(configfile, bool(options.daemonize)))
```

Comment:
I might be misreading this, but it seems to be under the `pid_running` condition now?